test/reline/test_reline.rb

```diff
@@ -217,11 +217,15 @@ def test_editing_mode
   end
 
   def test_readmultiline
-    # TODO
+    # readmultiline is module function
+    assert_equal(Reline.methods.include?(:readmultiline), true)
+    assert_equal(Reline.private_instance_methods.include?(:readmultiline), true)
   end
 
   def test_readline
-    # TODO
+    # readline is module function
+    assert_equal(Reline.methods.include?(:readline), true)
+    assert_equal(Reline.private_instance_methods.include?(:readline), true)
```

Comment:
The first argument of assertions should be "expected value" (`true`), or use `assert` method.

Reply:
Thanks!
I fixed it.